docs/Installation/Installation.md

```diff
@@ -179,8 +179,14 @@ To use Singularity you must:
    Singularity instructions on setting up additional [bind
    points](http://singularity.lbl.gov/docs-mount). Once inside the WORKDIR,
    clone SpECTRE into `WORKDIR/SPECTRE_ROOT`.
-3. Run `singularity build spectre.img
+3. Run `sudo singularity build spectre.img
    docker://sxscollaboration/spectrebuildenv:latest`.
+
+   If you get the error message that `makesquashfs` did not have enough space to
+   create the image you need to set a different `TMPDIR`. This can be done by
+   running: `sudo TMPDIR=/path/to/new/tmp singularity build spectre.img
+   docker://sxscollaboration/spectrebuildenv:latest`. Normally `TMPDIR` is
+   `/tmp`, but building the image will temporarily need almost 8GB of space.
```

Comment:
I also had to change `SINGULARITY_LOCALCACHEDIR` and `SINGULARITY_TMPDIR`, which also default to `/tmp`

Reply:
Okay, switched to the `SINGULARITY_TMPDIR`, and also added a discussion of `SINGULARITY_CACHEDIR` (this is what is used with Singularity 3)